biztalk/dev-center/mvp-and-community-blogs.md

```diff
@@ -28,7 +28,7 @@ Martin has been "doing" Integration for over 15 years, and is one of the admin t
 Ramblings about all things integration, including BizTalk Server, API Management, Visual Studio Team Services (VSTS), Service Bus, Logic Apps, and more.
```

Comment:
"Visual Studio Team Services (VSTS)" is now called "Azure DevOps". Fixed other branding issues.

```suggestion
Ramblings about all things integration, including BizTalk Server, API Management, Azure DevOps, Azure Service Bus, Azure Logic Apps, and more.
```